### PR TITLE
CHANGELOG: add release notes for v2.1.14

### PR DESCRIPTION
**Pull request by winkyao**
### What problem does this PR solve? 

add release notes for v2.1.14

@zz-jason @jackysp @GregoryIan @shenli PTAL

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,22 @@
 # TiDB Changelog
 All notable changes to this project will be documented in this file. See also [Release Notes](https://github.com/pingcap/docs/blob/master/releases/rn.md), [TiKV Changelog](https://github.com/tikv/tikv/blob/master/CHANGELOG.md) and [PD Changelog](https://github.com/pingcap/pd/blob/master/CHANGELOG.md).
 
+## [2.1.14] 2019-07-04
+* Fix wrong query results caused by column pruning in some cases [11019](https://github.com/pingcap/tidb/pull/11019)
```
Comment (jackysp) [inline CHANGELOG.md]: missing `#` for pr ID

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,22 @@
...
+* Fix the wrongly displayed information in `db` and `info` columns of `show processlist` [11000](https://github.com/pingcap/tidb/pull/11000)
+* Fix the issue that `MAX_EXECUTION_TIME` as a SQL hint and global variable does not work in some cases [10999](https://github.com/pingcap/tidb/pull/10999)
+* Support automatically adjust the incremental gap allocated by auto-increment ID based on the load [10997](https://github.com/pingcap/tidb/pull/10997)
```
Comment (jackysp) [inline CHANGELOG.md]:
```suggestion
* Support automatically adjust the incremental step allocated by auto-increment ID based on the load [10997](https://github.com/pingcap/tidb/pull/10997)
```

**Comment by winkyao** [comment]
@jackysp PTAL

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,22 @@
 # TiDB Changelog
 All notable changes to this project will be documented in this file. See also [Release Notes](https://github.com/pingcap/docs/blob/master/releases/rn.md), [TiKV Changelog](https://github.com/tikv/tikv/blob/master/CHANGELOG.md) and [PD Changelog](https://github.com/pingcap/pd/blob/master/CHANGELOG.md).
 
+## [2.1.14] 2019-07-04
+* Fix wrong query results caused by column pruning in some cases [#11019](https://github.com/pingcap/tidb/pull/11019)
+* Fix the wrongly displayed information in `db` and `info` columns of `show processlist` [11000](https://github.com/pingcap/tidb/pull/11000)
```
Comment (jackysp) [inline CHANGELOG.md]:
```suggestion
* Fix the wrongly displayed information in `db` and `info` columns of `show processlist` [#11000](https://github.com/pingcap/tidb/pull/11000)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,22 @@
...
+* Fix the issue that `MAX_EXECUTION_TIME` as a SQL hint and global variable does not work in some cases [10999](https://github.com/pingcap/tidb/pull/10999)
```
Comment (jackysp) [inline CHANGELOG.md]:
```suggestion
* Fix the issue that `MAX_EXECUTION_TIME` as a SQL hint and global variable does not work in some cases [#10999](https://github.com/pingcap/tidb/pull/10999)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,22 @@
...
+* Support automatically adjust the incremental step allocated by auto-increment ID based on the load [10997](https://github.com/pingcap/tidb/pull/10997)
```
Comment (jackysp) [inline CHANGELOG.md]:
```suggestion
* Support automatically adjust the incremental step allocated by auto-increment ID based on the load [#10997](https://github.com/pingcap/tidb/pull/10997)
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,6 +1,22 @@
...
+* Fix the issue that the `Distsql` memory information of `MemTracker` is not correctly cleaned when a query ends [10971](https://github.com/pingcap/tidb/pull/10971)
```
Comment (jackysp) [inline CHANGELOG.md]:
```suggestion
* Fix the issue that the `Distsql` memory information of `MemTracker` is not correctly cleaned when a query ends [#10971](https://github.com/pingcap/tidb/pull/10971)
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,6 +1,22 @@
...
+* Add the `MEM` column in the `information_schema.processlist` table to describe the memory usage of a query [10896](https://github.com/pingcap/tidb/pull/10896)
```
Comment (jackysp) [inline CHANGELOG.md]:
```suggestion
* Add the `MEM` column in the `information_schema.processlist` table to describe the memory usage of a query [#10896](https://github.com/pingcap/tidb/pull/10896)
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,6 +1,22 @@
...
+* Add the `max_execution_time` global system variable to control the maximum execution time of a query [10940](https://github.com/pingcap/tidb/pull/10940)
```
Comment (jackysp) [inline CHANGELOG.md]:
```suggestion
* Add the `max_execution_time` global system variable to control the maximum execution time of a query [#10940](https://github.com/pingcap/tidb/pull/10940)
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,6 +1,22 @@
...
+* Fix the panic caused by using unsupported aggregate functions [10911](https://github.com/pingcap/tidb/pull/10911)
```
Comment (jackysp) [inline CHANGELOG.md]:
```suggestion
* Fix the panic caused by using unsupported aggregate functions [#10911](https://github.com/pingcap/tidb/pull/10911)
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,6 +1,22 @@
...
+* Add an automatic rollback feature for the last transaction when the `load data` statement fails [10862](https://github.com/pingcap/tidb/pull/10862)
```
Comment (jackysp) [inline CHANGELOG.md]:
```suggestion
* Add an automatic rollback feature for the last transaction when the `load data` statement fails [#10862](https://github.com/pingcap/tidb/pull/10862)
```

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,6 +1,22 @@
 # TiDB Changelog
 All notable changes to this project will be documented in this file. See also [Release Notes](https://github.com/pingcap/docs/blob/master/releases/rn.md), [TiKV Changelog](https://github.com/tikv/tikv/blob/master/CHANGELOG.md) and [PD Changelog](https://github.com/pingcap/pd/blob/master/CHANGELOG.md).
 
+## [2.1.14] 2019-07-04
+* Fix wrong query results caused by column pruning in some cases [#11019](https://github.com/pingcap/tidb/pull/11019)
+* Fix the wrongly displayed information in `db` and `info` columns of `show processlist` [#11000](https://github.com/pingcap/tidb/pull/11000)
+* Fix the issue that `MAX_EXECUTION_TIME` as a SQL hint and global variable does not work in some cases [#10999](https://github.com/pingcap/tidb/pull/10999)
```
Comment (GregoryIan) [inline CHANGELOG.md]:
```suggestion
* Fix the issue that `MAX_EXECUTION_TIME` as a SQL hint and global variable does not work under some cases [#10999](https://github.com/pingcap/tidb/pull/10999)
```

**Comment by GregoryIan** [comment]
Rest LGTM

**Comment by winkyao** [comment]
@GregoryIan PTAL

**Pull request merged**